src/main/java/wordle/model/LetterCounter.java

```diff
@@ -0,0 +1,57 @@
+package wordle.model;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LetterCounter {
+
+    private final Map<Character, Integer> letterCountMap;
+
+    public LetterCounter(Letters letters) {
+        this.letterCountMap = new HashMap<>();
+        for (Letter answerLetter : letters.getLetters()) {
+            char value = answerLetter.getValue();
+            letterCountMap.put(value, letterCountMap.getOrDefault(value, 0) + 1);
+        }
+    }
```

Comment:
`Letters` 는 `Letter` 제네릭 타입의 1급 컬렉션으로 보여지네요!👍🏻
저는 개인적으로 외부에서 `for` 문을 사용하는 1급 컬렉션이라면 `Iterable` 를 구현하는 방법도 좋다고 생각하는데 어떻게 생각하시나요?

```suggestion
    public LetterCounter(Letters letters) {
        this.letterCountMap = new HashMap<>();
        letters.forEach(answerLetter -> {
            char value = answerLetter.getValue();
            letterCountMap.put(value, letterCountMap.getOrDefault(value, 0) + 1);
        });
        // 또는
        for (Letter answerLetter : letters) {
            char value = answerLetter.getValue();
            letterCountMap.put(value, letterCountMap.getOrDefault(value, 0) + 1);
        }
    }
```

Reply:
오 이런방법이...... 👍 감사합니다. 몰랐는데, 적용하니까 훨씬 깔끔해지네요! 회사 코드에도 적용했습니다 ㅋㅋㅋㅋ 😄 꿀팁 너무 감사합니다.  

추가로 궁금한게 있는데, 혹시 `stream()`도 구현해서 사용하시는 편이실까요? 회사 코드에는 `stream()`을 애용하고 있어서 Iterable로 만듦으로써 사용 가능해지는 forEach()나 향상된 for문은 자주 사용하지 않거든요!

이를 여쭙는 이유는, `내가 미처 생각하지 못한 주의사항이 있을까?`라는 노파심에 여쭤봅니다 ㅎㅎ 혹시 없다면 회사 코드에도 적용해보려고 해요 ㅎㅎ